apps/frontend/src/components/ui/RangeDatePicker.tsx

```
@@ -0,0 +1,79 @@
+                  )
+                ) : (
+                  'Pick a date range'
+                )}
+              </span>
+              {date && (
+                <div
+                  onClick={handleClear}
+                  className="absolute flex items-center justify-center w-6 h-6 text-foreground/30 hover:text-foreground cursor-pointer rounded-sm hover:bg-popover-foreground/10 right-13 transition-all"
+                  tabIndex={0}
+                  aria-label="Clear input"
+                >
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)